internal/operators/inspect_file.go

```diff
@@ -32,11 +32,11 @@ func (o *inspectFile) Evaluate(tx plugintypes.TransactionState, value string) bo
 	defer cancel()
 	// Add /bin/bash to context?
 	cmd := exec.CommandContext(ctx, o.path, value)
-	_, err := cmd.CombinedOutput()
+	output, err := cmd.CombinedOutput()
 	if ctx.Err() == context.DeadlineExceeded || err != nil {
 		return false
 	}
-	return true
+	return len(output) > 0 && output[0] != '1'
```

Comment:
What is this condition about `output[0] != '1'`?

Reply:
I can confirm this is the expected behavior.

Reply:
If this is the expected behaviour we should have reference documentation in coraza to back it up

Reply:
It is not well documented, the references I used while writing inspectFile were old CRS files, you can look at the clamav integration
We should indeed add it to our docs